pytorch_lightning/core/optimizer.py

```diff
@@ -11,38 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import collections
-import copy
-import inspect
-import os
-import re
-import tempfile
 import types
-from abc import ABC
-from argparse import Namespace
-from typing import Any, Callable, Dict, List, Mapping, Optional, Sequence, Tuple, Union
+from typing import Any, Callable, Optional
 from weakref import proxy
 
-import torch
-from torch import ScriptModule, Tensor
-from torch.nn import Module
-from torch.optim import SGD
 from torch.optim.optimizer import Optimizer
 
-from pytorch_lightning import _logger as log
-from pytorch_lightning.callbacks import Callback
-from pytorch_lightning.core.grads import GradInformation
-from pytorch_lightning.core.hooks import CheckpointHooks, DataHooks, ModelHooks
-from pytorch_lightning.core.memory import ModelSummary
-from pytorch_lightning.core.saving import ALLOWED_CONFIG_TYPES, PRIMITIVE_TYPES, ModelIO
```

Comment:
where did this go? cc @tchaton

Reply:
I guess useless imports.

Reply:
shall not be there at all in the first place, @SeanNaren